src/templates/invoice/Invoice.tsx

```diff
@@ -0,0 +1,160 @@
+import { css } from "@emotion/core";
+import { TemplateProps } from "@govtechsg/decentralized-renderer-react-components";
+import React, { FunctionComponent } from "react";
+import { Invoice } from "./types";
+import { format } from "date-fns";
+
+const container = css`
+  font-family: "Lucida Sans Unicode", "Lucida Grande", sans-serif;
+  color: #4e4e50;
+  width: 100%;
+  height: 100vh;
+  max-height: 1000px;
+  max-width: 980px;
+  position: relative;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+`;
```

Comment:
i not sure if u need this? u copied it over from coveringLetter yeah? u may want to tweak